lib/tasks/translate_test.rake

```diff
@@ -57,4 +57,12 @@ namespace :translate_test do # rubocop:disable Metrics/BlockLength
   task unsubscribe: :environment do
     Rails.cache.write(Force::EventSubscriberTranslateService::UNSUBSCRIBE_CACHE_KEY, true)
   end
+
+  task flag: :environment do
```

Comment:
suggestion: Also add a `TODO` comment here.

Reply:
I actually didn't mean to leave this in here - it doesn't work